.gitattributes

```diff
@@ -0,0 +1 @@
+data/library_coverage_xml_and_fulltext_indicators.db.xz filter=lfs diff=lfs merge=lfs -text
```

Comment:
Sure, I'm fine with that. To confirm, though, that would also include the TSV file, which currently is also saved with a `.tsv.xz` extension (saving it without was causing LibreOffice to try to open it directly on my system, resulting in garbled-looking data). I remember you mentioning that the data file itself shouldn't use LFS. Is it ok with you to use LFS for both?

Reply:
If a file is compressed, it should have the extension for its compression. If this is the case, not sure why LibreOffice would try to open your XZ (`*.tsv.xz`) compressed file.

If a file is compressed, it should be tracked using git LFS.

If you want line-by-line diffing for a file, it should not be compressed but rather plain text.

Does this answer your questions?

Reply:
I meant that I was originally compressing the TSV but not adding the `.xz` extension, and LibreOffice would try to open that. So I added the `.xz` extension, which now correctly gets opened in my system's archive manager program, which shows the uncompressed file that LibreOffice can successfully open.

So yes, that does answer my question: since the TSV *is* compressed, I'll track it with LFS, too, using the edit you suggested.

Reply:
Implemented in a09dc9.

Reply:
@publicus, when writing commit hashes in GitHub, I'd recommend using at least 7 characters. For example, `a09dc99 versus a09dc9` is written below:

a09dc99 versus a09dc9. GitHub needs at least 7 characters to auto-link.

> Implemented in a09dc9

Notices such as above are helpful, although not essential... so do it only if it helps you organize things.